.gitignore

```diff
@@ -59,7 +59,8 @@ lib
 temp/*	
 tests/casperjs/*
 report/*
-src/.DS_Store
+.DS_Store/
+src/.DS_Store/
```

Comment:
In theory these 2 entries [shouldn't be necessary](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository#comment15789759_107921).

File `src/.DS_Store` should be removed from the history: if you check the diff you will see it has been committed.